packages/cozy-client/src/helpers.js

```diff
@@ -76,6 +78,34 @@ export const generateWebLink = ({
   return url.toString()
 }
 
+/**
+ * Generate Sharing link for one or many files
+ *
+ * @param {CozyClient} client - Instance of CozyClient
+ * @param {string[]} filesIds - Array of io.cozy.files ids
+ * @param {boolean} [isFlatDomain] -
+ * @returns {Promise<string>} Shared link
+ */
+export const getSharingLink = async (client, filesIds, isFlatDomain) => {
+  const PERMS = {
+    _type: DOCTYPE_PERMISSIONS,
+    permissions: {
+      files: { type: DOCTYPE_FILES, values: filesIds, verbs: ['GET'] }
+    }
+  }
+  const { data: sharedLink } = await client.save(PERMS)
+
+  const webLink = generateWebLink({
+    cozyUrl: client.getStackClient().uri,
+    searchParams: [['sharecode', sharedLink?.attributes?.shortcodes?.code]],
+    pathname: '/public',
+    slug: 'drive',
+    subDomainType: isFlatDomain ? 'flat' : 'nested'
+  })
+
+  return webLink
+}
+
```

Comment:
can you move it to a new file called sharing.js(https://github.com/cozy/cozy-client/tree/master/packages/cozy-client/src/models) ?

Reply:
Ok, that's better indeed!
It's done

Reply:
can you remove this commit ? :p

Reply:
:face_palm: